recipes/deeppurpose/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "deeppurpose" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kexinhuang12345/DeepPurpose/archive/v{{ version }}.tar.gz
+  sha256: 3090375c9fca73ff24c4e0c2c82f0c3fca7c96e77eac66ba4ab5f84f28697a1c
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
If this package really is `noarch:python` (and it seems to be pure python), then you shouldn't need the`build` section or anything in it.

Reply:
Hi, thanks! Just removed this section.